src/chordSymbol.js

```diff
@@ -0,0 +1,593 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
```

Comment:
Please lowercase the filename (to stay consistent with other files.)

Reply:
done

Reply:
Did you push the update? (I still see `chordSymbol.js`)

Reply:
I did.  I'm guessing this is just the github review widget.  When I pull the fork, it is lower case and it won't even compile otherwise.

Reply:
well, hang on I take that back.  It does not show up but I did make the change.  Let me look into it.

Reply:
ok, live and learn.  I changed it in GH site and it seems good now.